templates/shared/actions/runner_list.tmpl

```diff
@@ -52,6 +52,7 @@
 					<th>{{.locale.Tr "actions.runners.owner_type"}}</th>
 					<th>{{.locale.Tr "actions.runners.labels"}}</th>
 					<th>{{.locale.Tr "actions.runners.last_online"}}</th>
+					<th>{{.locale.Tr "actions.runners.edit"}}</th>
 					<th></th>
```

Comment:
There is an empty `<th>` below, this column is left as empty by design, I do not think it needs a "column title"

Reply:
Ah, fogot to remove it.

ps: The root reason of this PR is that I found that there's a word break of `Last Online Time` which seems strange.
![image](https://user-images.githubusercontent.com/18380374/234437353-f74292ee-308d-4346-bc5b-53f3e874369b.png)

Then I found that in other pages, `edit` column has a title.
![image](https://user-images.githubusercontent.com/18380374/234437154-c8632f40-9ce1-420b-9e39-e24e0d07a12b.png)
![image](https://user-images.githubusercontent.com/18380374/234437177-bc429215-54c8-4f59-b5bd-4b5d5ef8cef7.png)